### ISSUE TITLE
Create OSDI'12 tag

### ISSUE THREAD

**Issue by rodrigosiqueira** [project-dune/dune#14]
Hi,
I noticed that Dune project does not have a tag related to OSDI'12. I would like to suggest the creation of OSDI'12 tag for keeping the results published on that conference replicable. Maybe the current state of master reflects it. I believe this can benefit the project since it is possible to manage the new contributions as an evolution from the original idea.

**Comment by kkaffes** [comment]
Since I just recently started working on Dune, I am not sure which version was used for OSDI'12. However, if a non-incremental change is about to be introduced, we can create a tag then.

**Comment by rodrigosiqueira** [comment]
Personally, I believe that is a good idea to keep any published result replicable for scientific reasons. Maybe you can create more than one tag, for example, one for OSDI and another of the current version working. I just have two points in favor to create a tag:
1. Create a tag on a version fully working is important for getting new users.
2. The project has to keep its evolution, and tags provide checkpoints.

Anyway, this is just a suggestion.

**Comment by kkaffes** [comment]
I agree with you. The main problem is that I do not know where exactly to set OSDI'12 tag.

**Comment by rodrigosiqueira** [comment]
OSDI'12 happened on 2012 October 8 (https://www.usenix.org/conference/osdi12), which means the call for paper closed around July or August. So, I guess the final version was in somewhere close to commit 3b633026a01dec392baec3189a2fd15f87b7b4b4.

If it is not possible to track it, I suggest to create a tag of a working version only.
